app/components/blog-post-preview.js

```diff
@@ -1,21 +1,21 @@
-import classic from 'ember-classic-decorator';
-import { computed } from '@ember/object';
-import { inject as service } from '@ember/service';
-import Component from '@ember/component';
+import classic from "ember-classic-decorator";
+import { computed } from "@ember/object";
+import { inject as service } from "@ember/service";
+import Component from "@ember/component";
```

Comment:
I accidentally forgot to turn off my editor's "format on save" and...there wound up being a bunch of formatting in these commits. I'll comment where things actually changed 😐

Reply:
Where does this come from exactly? Should we check in some file that enforces these?

Reply:
Yeah -- this is coming from [prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode). We could add [`eslint-plugin-prettier`](https://github.com/prettier/eslint-plugin-prettier) along with [prettier](https://prettier.io/docs/en/install.html) to make it part of the linting workflow. We could also use [husky](https://github.com/typicode/husky) to automatically run prettier before every commit, for folks who don't have format-on-save in their editor.